nitro-protocol/hardhat-deploy/01-token.ts

```diff
@@ -8,14 +8,6 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
     getNamedAccounts,
   } = hre;
 
-  const chainId = await getChainId();
-
-  // This is the default hardhat network chainid
-  if (chainId !== '31337') {
-    console.info(`Skipping deployment of Token for chain with ID: ${chainId}`);
-    return;
-  }
-
```

Comment:
This code restricts deployment of the token contract to the Hardhat network. Since we are not targeting mainnet at the moment (and costs are not a concern with testnets), it is less error prone to deploy all contracts to a network. A scenario that is easy to envision is:
1. We switch to a new testnet.
2. We trigger contract deploys.
3. Testnet tests do not pass since one of the contracts is not deployed.
4. Developer time is used to debug and understand the underlying cause.

We do want to be careful to deploy only the minimum necessary contracts when deploying to mainnet.